op-node/withdrawals/utils.go

```diff
@@ -265,7 +265,7 @@ func ParseMessagePassed(receipt *types.Receipt) (*bindings.L2ToL1MessagePasserMe
 // StorageSlotOfWithdrawalHash determines the storage slot of the Withdrawer contract to look at
 // given a WithdrawalHash
 func StorageSlotOfWithdrawalHash(hash common.Hash) common.Hash {
-	// The withdrawals mapping is the second (0 indexed) storage element in the Withdrawer contract.
+	// The withdrawals mapping is the 0th storage slot in the L2ToL1MessagePasser contract.
```

Comment:
Mapping is also called `sentMessages` (https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/.storage-layout#L157)